flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -274,7 +274,10 @@ protected static Notification createNotification(
                     : notificationDetails.body)
             .setTicker(notificationDetails.ticker)
             .setAutoCancel(BooleanUtils.getValue(notificationDetails.autoCancel))
-            .setContentIntent(pendingIntent)
+            .setContentIntent(
+                BooleanUtils.getValue(notificationDetails.selectable) 
```

Comment:
Compared to other parts of the code, I don't believe you should be using `BooleanUtils.getValue()`. That is for coalescing null values to false. The null values would happen when introducing new fields/properties and there are scheduled notifications kept in shared preferences. With the changes introduced in the PR, my understanding is that null `selectable` value be coalesced to `true` instead of `false`